src/amo/components/AddonMoreInfo/index.js

```diff
@@ -149,8 +172,34 @@ export class AddonMoreInfoBase extends React.Component<InternalProps> {
       );
     }
 
+    const comma = i18n.gettext(',');
+    const renderRelatedCategories = [];
+    const separator = isRTL ? ` ${comma}` : `${comma} `;
```

Comment:
I think the CSS approach taken by @eviljeff for tags is superior here (see #10798). Translating only a comma is going to be difficult for localizers and we can see above that RTL mode does not make it easier.